Core/Product Request/PaymentTransactionObserver.swift

```diff
@@ -79,19 +79,23 @@ public class PaymentTransactionObserver: NSObject, SKPaymentTransactionObserver
     }
     
     public func paymentQueueRestoreCompletedTransactionsFinished(_ queue: SKPaymentQueue) {
-        // Check to see if the returned queue as any transactions.  If there are no
-        // transactions, it means the user has not made any purchases yet, so there is
-        // nothing to restore.
-        guard queue.transactions.count > 0 else {
-            // There were no transactions for this user
-            let localizedErrorMessage = NSLocalizedString("Sorry, no purchased products were found. If you have questions, please contact support@blackpixel.com.", tableName: nil, bundle: Bundle(for: PaymentTransactionObserver.self), value: "", comment: "no previous transactions were found on the users current iTunes account error message")
-            
-            delegate?.transactionDidFail(localizedErrorMessage: localizedErrorMessage)
+        // fetch support email or nil if it doesn't exist
+        if let info = Bundle.main.infoDictionary,
```

Comment:
This will load the Info.plist from `IAP.framework`'s bundle. I don't see the HGSupportEmail key there; is it added elsewhere?

Reply:
incorrect, this loads from the main app bundle as verified in the debugger.

Reply:
I'm pretty sure @loganjones is right.

Reply:
Nevermind. I see what is going on.

Reply:
I must be confused with another issue; I know I've had trouble in the past with `Bundle.main` not giving me what I thought it would. It's weird because the documentation could be interpreted multiple ways:
> For code running in a framework, the main bundle offers access to the framework’s bundle directory.
https://developer.apple.com/documentation/foundation/bundle/1410786-main